themes/digital.gov/src/js/gh-edit-commit.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Sets the most recent commit timestamp for the page
+ * This is displayed in #page-data section at the bottom of the page
+ */
+
+// eslint-disable-next-line func-names
+(function () {
+  const editPageLink = document.querySelector("#page-data .gh-commit-date");
+  const gitRepo = {
+    filepath: "",
+    branch: null,
+  };
+
+  /**
+   * Get hugo file path from the data-gh-edit-page attribute to link to Github repo location
+   * example string: news/2023/07/2023-07-19-gsa-shared-service-provider-program-guide.md
+   */
+  function getFilepath() {
+    if (editPageLink) {
+      gitRepo.filepath = document
+        .querySelector("div[data-gh-edit-page]")
```

Comment:
Consider a less specific selector, if we wanted to move it to `article` or `main` this wouldn't work.

```suggestion
        .querySelector("[data-gh-edit-page]")
```

Reply:
Great point, selector is updated!